packages/meteor-gazelle-wiki/lib/routes.js

```diff
@@ -0,0 +1,60 @@
+WikiController = ApplicationController.extend({
+  waitOn: function () {
+    return Meteor.subscribe('announcements');
+  },
+  data: function () {
+    return {
+      wiki: Articles.find({}, { sort: { createdAt: -1 }})
```

Comment:
Probably don't want to send all articles to the client at once, but we can come back to this later.